packages/v4/package.json

```diff
@@ -20,6 +20,7 @@
     "@patternfly/documentation-framework": "^1.3.0",
     "@patternfly/quickstarts": "^2.2.4",
     "@patternfly/react-docs": "5.103.2",
+    "@ausuliv/react-catalog-view-extension": "1.1.0",
```

Comment:
I'm currently releasing the standalone catalog view package under my own NPM account for testing purposes. I'll replace this with the scope here with `@patternfly` before we actually merge this PR.